src/structures/Message.ts

```diff
@@ -39,12 +42,24 @@ export class Message extends Base {
   public async delete(options: MessageDeleteOptions): Promise<Message> {
     return new Promise((resolve, reject) => {
       setTimeout(async () => {
-        return await this.client.api.request({
+        await this.client.api.request({
           method: "DELETE",
-          endpoint: `channels/channel/messages/${this.id}`,
+          endpoint: `channels/channelID/messages/${this.id}`,
```

Comment:
This channelID will need to be replaced. Need to add a property called channel that will have a typeof text / dm / newschannel.  essentially

```ts
private channel: TextChannel | DMChannel | NewsChannel; 

//down at the delete method
endpoint: `channels/${this.channel.id}/messages/${this.id}`
```

Reply:
Ik, that's why I left it like that as we don't have a channel structure yet, we'll do that once we have the structure.